src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -335,18 +335,13 @@ private UserTenantEntity toEntity(UserTenant userTenantDto, UUID consortiumId, U
 
   private PermissionUser createPermissionUser(String userId) {
     List<String> emptyPermissionList = new ArrayList<>();
-    Optional<PermissionUser> permissionUserOptional = permissionsClient.get("userId==" + userId)
-      .getPermissionUsers()
-      .stream()
-      .findFirst();
+    Optional<PermissionUser> permissionUserOptional = permissionService.getPermissionUserById(userId);
```

Comment:
permissionUser and user have different ids, correct? if so - please rename method to: getPermissionUserByUserId

Reply:
or even lets rename PermissionService to PermissionUsersServcie and lets delete words permissionUser from each method name, so we will have
getById()
getByUserId()
...

Reply:
Yes Correct! fixed